support-frontend/assets/pages/paper-subscription-checkout/components/thankYou.jsx

```diff
@@ -32,53 +33,68 @@ import { type FormFields, getFormFields } from 'helpers/subscriptionsForms/formF
 };
 
+// ----- Map State/Props ----- //
+
+function mapStateToProps(state: WithDeliveryCheckoutState) {
+  return {
+    ...getFormFields(state),
+    useDigitalVoucher: state.common.settings.useDigitalVoucher,
+  };
+}
 
 // ----- Component ----- //
 
-const whatNext: {[FulfilmentOptions]: Element<*>} = {
-  [HomeDelivery]: (
-    <Text title="What happens next?">
-      <OrderedList items={[
-        <span>
-          Look out for an email from us confirming your subscription.
-          It has everything you need to know about how manage it in the future.
-        </span>,
-        <span>
-          Your newspaper will be delivered to your door.
-        </span>,
-        ]}
-      />
+const whatNextText: { [FulfilmentOptions]: { [key: string]: Array<string> } } = {
+  [HomeDelivery]: {
+    default: [
+      `Look out for an email from us confirming your subscription.
+        It has everything you need to know about how manage it in the future.`,
+      'Your newspaper will be delivered to your door.',
+    ],
+  },
+  [Collection]: {
+    default: [
+      `Look out for an email from us confirming your subscription.
+        It has everything you need to know about how to manage it in the future.`,
+      'You will receive your personalised book of vouchers.',
+      'Exchange your voucher for a newspaper at your newsagent or wherever you buy your paper',
+    ],
+    digitalVoucher: [
+      `Keep an eye on your inbox. You should receive an email confirming the details of your subscription,
+        and another email shortly afterwards that contains details of how you can pick up your papers from today!`,
+      `You will receive your Subscription Card in your subscriber pack in the post, along with your home
+        delivery letter.`,
+      `Visit your chosen participating newsagent to pick up your paper using your Subscription Card, or
+        arrange a home delivery using your delivery letter.`,
+    ],
+  },
+};
 
-    </Text>
-  ),
-  [Collection]: (
+function whatNextElement(textItems) {
+  return (
     <Text title="What happens next?">
       <p>
-        <OrderedList items={[
-          <span>
-            Look out for an email from us confirming your subscription.
-            It has everything you need to know about how to manage it in the future.
-          </span>,
-          <span>
-            You will receive your personalised book of vouchers.
-          </span>,
-          <span>
-            Exchange your voucher for a newspaper at your newsagent or wherever you buy your paper
-          </span>,
-          ]}
-        />
-
+        <OrderedList items={textItems.map(item => <span>{item}</span>)} />
       </p>
     </Text>
-  ),
-};
+  );
+}
+
+function WhatNext(fulfilmentOption, useDigitalVoucher = false) {
+  if (fulfilmentOption === Collection && useDigitalVoucher) {
+    return whatNextElement(whatNextText[Collection].digitalVoucher);
+  }
+  return whatNextElement(whatNextText[fulfilmentOption].default);
```

Comment:
minor but the if and else part of this both repeat the call to `whatNextElement(` so it would be a good habit to put only the bits that change in the if statement.